submissions/SpidertronWeaponSwitcher/SpidertronWeaponSwitcher.md

```diff
@@ -4,21 +4,21 @@ author: Xorimuth
 
 ### Under the hood: [Spidertron Weapon Switcher](https://mods.factorio.com/SpidertronWeaponSwitcher)
 
-![Demonstration of switching between weapons](https://github.com/Xorimuth/Alt-F4/blob/master/submissions/SpidertronWeaponSwitcher/SWS-demo-gif.gif?raw=true)
+Spidertron Weapon Switcher is a mod that allows Spidertron to fire any kind of weapon. In this post, I will be talking about how it works behind the scenes. But first, a peak at how awesome this looks in action:
 
-Spidertron Weapon Switcher is a mod that allows a spidertron to fire any weapon. In this post, I will be talking about how it works behind the scenes.
+![Demonstration of switching between weapons](https://github.com/Xorimuth/Alt-F4/blob/master/submissions/SpidertronWeaponSwitcher/SWS-demo-gif.gif?raw=true)
 
-If you’ve never written a mod before, you may not know that many things in Factorio cannot be changed _during the game_, for example, the range of a gun, the graphics of a building and, importantly for me, the weapon slots that a vehicle has. Of course, the weapons that a spidertron has _can_ be changed by mods, but only by creating a new ‘type’ of spidertron during the game’s loading phase.
+If you’ve never written a mod before, you may not know that many things in Factorio cannot be changed _during the game_, like for example the range of a gun, the graphics of a machine and, importantly for me, the weapon slots that a vehicle has. Of course, the weapons that Spidertron has _can_ be changed by mods, but only by creating a new ‘type’ of Spidertron during the loading phase of the game.
```

Comment:
In general I tried to use the same grammar style for 'spidertron' as [the wiki](https://wiki.factorio.com/Spidertron) does, i.e. "the spidertron" or "a spidertron" vs simply "Spidertron".

I don't feel strongly about this though.

Perhaps you were trying to differentiate between "a spidertron" and "the definition of a spidertron", which I quite like, but I think that the first time I said "a spidertron" should stay as I wrote it.

Reply:
I agree with @Xorimuth, matching the wiki (which consistently uses lowercase) feels like the right choice to me.